game/cards/dm06/light_bringer.go

```diff
@@ -0,0 +1,102 @@
+package dm06
+
+import (
+	"duel-masters/game/civ"
+	"duel-masters/game/family"
+	"duel-masters/game/fx"
+	"duel-masters/game/match"
+)
+
+func AdomisTheOracle(c *match.Card) {
+
+	c.Name = "Adomis, the Oracle"
+	c.Power = 2000
+	c.Civ = civ.Light
+	c.Family = []string{family.LightBringer}
+	c.ManaCost = 3
+	c.ManaRequirement = []string{civ.Light}
+	c.TapAbility = true
+
+	c.Use(fx.Creature, fx.When(fx.TapAbility, func(card *match.Card, ctx *match.Context) {
+
+		shields := fx.SelectBackside(
+			card.Player,
+			ctx.Match,
+			card.Player,
+			match.SHIELDZONE,
+			"Adomis, the Oracle: Select one of your shields that will be shown to you",
+			1,
+			1,
+			true,
+		)
+
+		ids := make([]string, 0)
+
+		for _, s := range shields {
+			ids = append(ids, s.ImageID)
+		}
+
+		ctx.Match.ShowCards(
+			card.Player,
+			"Your shield:",
+			ids,
+		)
+
+		card.Tapped = true
+
+	}))
+}
```

Comment:
I think the card text means it should be possible to choose a card from the opponent's shieldzone too, unless there's some rulings i'm not aware of

Reply:
Oh, it shoud actually, but with that i dont know if i can make it work